Automattic/woocommerce-payments
# feat: additional payment methods onboarding

Fixes https://github.com/Automattic/woocommerce-payments/issues/1633

#### Changes proposed in this Pull Request









UI changes on the onboarding steps.
Data implementation is done separately: https://github.com/Automattic/woocommerce-payments/issues/1735

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Go to your WC Home: http://localhost:8082/wp-admin/admin.php?page=wc-admin
- Click on the "Set up additional payment methods" task, under "Extensions setup": https://d.pr/i/yLQYTc (please note: this section is called/appears differently in different versions of WC)
- On the first step, click "Continue" (you don't need to select anything) - https://d.pr/i/tCMdie
- Now the first step is marked as "completed" - the second step appears and you can go to the WC home: https://d.pr/i/x8dYGS
- The task is now marked as "completed": https://d.pr/i/BDm86S

-------------------
